### PR TITLE
Feature/632 SB header changes

### PR DESCRIPTION
**Pull request by john-labbate**
Smart Bulletin Header Changes
[#632](https://github.com/GSA/smartpay-website/issues/632)

### CODE REVIEW

File: src/content/state-taxes/louisiana.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ updated: 2023-02-15
 
 ### Travel Card
 
-* [R-1376 Government Employees Hotel Lodging Sales/Use Tax Exemption Certificate](https://revenue.louisiana.gov/Forms/ForBusinesses)
+* [R-1376 Government Employees Hotel Lodging Sales/Use Tax Exemption Certificate [PDF, 2 pages]](/files/R_1376_Louisiana_State_Tax_Travel_Form.pdf)
```
Comment (CodyHinze) [inline src/content/state-taxes/louisiana.md:19]: This should be removed from this PR, This change is not desired.  See PR 695.

File: src/content/smart-bulletins/036.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
-title: "Smart Bulletin No. 036"
-intro: "Use of GSA SmartPay® Charge Cards to Pay for Electric Vehicle Charging Pursuant to the “Charging Helps Agencies Realize General Efficiencies Act” or “CHARGE Act”).."
+title: "Use of GSA SmartPay® Charge Cards to Pay for Electric Vehicle Charging Pursuant to the “Charging Helps Agencies Realize General Efficiencies Act” or “CHARGE Act”).."
```
Comment (CodyHinze) [inline src/content/smart-bulletins/036.md:2]: Please remove the ").." per meeting with John B.

File: src/content/smart-bulletins/034.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ---
-title: "Smart Bulletin No. 034"
-intro: "Requests for JPMorgan Chase & Co. (JPM) GSA SmartPay 2® (SP2) Data"
+title: "Requests for JPMorgan Chase & Co. (JPM) GSA SmartPay 2® (SP2) Data"
```
Comment (CodyHinze) [inline src/content/smart-bulletins/034.md:2]: Should be "GSA SmartPay® 2".

File: src/content/state-taxes/louisiana.md (hunk 1 above)
Comment (john-labbate) [reply src/content/state-taxes/louisiana.md]: Updated sorry started from a branch that wasnt staging

File: src/content/smart-bulletins/034.md (hunk 3 above)
Comment (john-labbate) [reply src/content/smart-bulletins/034.md]: updated

File: src/content/smart-bulletins/033.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
 ---
-title: "Smart Bulletin No. 033"
-description: "Smart Bulletin 033 provides guidance to agencies implementing section 889(a)(1)(B) of the John S. McCain National Defense Authorization Act (NDAA). The act prohibits Federal agencies from purchasing products or services from prohibited vendors. You can find a list of prohibited vendors and learn more about how this affects GSA SmartPay program participants."
+title: "Smart Bulletin 033 provides guidance to agencies implementing section 889(a)(1)(B) of the John S. McCain National Defense Authorization Act (NDAA). The act prohibits Federal agencies from purchasing products or services from prohibited vendors. You can find a list of prohibited vendors and learn more about how this affects GSA SmartPay program participants."
```
[the remark this answers is not in the document]
Comment (john-labbate) [reply src/content/smart-bulletins/033.md]: updated

File: src/content/smart-bulletins/036.md (hunk 2 above)
Comment (john-labbate) [reply src/content/smart-bulletins/036.md]: updated

**Review by CodyHinze — APPROVED** [review]
All content and requested changes approved.

**Pull request merged**